daemon/c_seccomp.c

```diff
@@ -123,6 +124,10 @@
 #endif
 // clang-format on
 
+#ifndef MODULE_INIT_COMPRESSED_FILE
+#define MODULE_INIT_COMPRESSED_FILE 4
+#endif
```

Comment:
shall we always rely on this value or include linux/module.h explicitly with this as fallback?

Reply:
yes will include the header